src/robotide/editor/__init__.py

```diff
@@ -25,24 +25,37 @@
 
 _EDIT = """
 [Edit]
+---
 &Undo | Undo last modification | Ctrlcmd-Z
 &Redo | Redo modification | Ctrlcmd-Y
 ---
-Cu&t | Cut | Ctrlcmd-X
-&Copy | Copy | Ctrlcmd-C
-&Paste | Paste | Ctrlcmd-V
-&Insert | Insert | Shift-Ctrl-V
-&Delete | Delete  | Del
+Make Variable | Make variable | Ctrlcmd-1
+Make List Variable | Make list variable | Ctrlcmd-2
+Make Dict Variable | Make dict variable | Ctrlcmd-5
 ---
-Comment | Comment selected rows | Ctrlcmd-3
-CommentCells | Comment cells with # | Ctrlcmd-Shift-3
-Uncomment | Uncomment selected rows | Ctrlcmd-4
-UncommentCells | Uncomment cells with # | Ctrlcmd-Shift-4
+Comment Cells | Comment selected cells with # | Ctrlcmd-Shift-3
+Uncomment Cells | Uncomment selected cells with # | Ctrlcmd-Shift-4
+Move Cursor Down | Move cursor down | Alt-Enter
+---
+Comment Rows | Comment selected rows | Ctrlcmd-3
+Uncomment Rows | Uncomment selected rows | Ctrlcmd-4
+Move Rows Up | Move selected rows up | Alt-Up
+Move Rows Down | Move selected rows down | Alt-Down
+Swap Row Up | Swap selected row with the one above | Ctrlcmd-T
 ---
 Insert Cells | Insert Cells | Shift-Ctrl-I
 Delete Cells | Delete Cells | Shift-Ctrl-D
 Insert Rows | Insert Rows | Ctrlcmd-I
 Delete Rows | Delete Rows | Ctrlcmd-D
+---
+Select All | Select All | Ctrlcmd-A
+---
+Cu&t | Cut | Ctrlcmd-X
+&Copy | Copy | Ctrlcmd-C
+&Paste | Paste | Ctrlcmd-V
```

Comment:
These functions are now in the bottom of a long list of functions. They should have stayed on the top.

Reply:
They are also at the bottom of the grid cell menu. Again following the rule: same content, same order